orchestrator.py

```diff
@@ -0,0 +1,35 @@
+from typing import Dict
+from game import Game
+from exceptions import GameAlreadyExistError
+
+class Orchestrator:
+  """
+  ## The Grand O
+  - Manages **EVERY** game initiated on Telegram.
+  - Communicates with players globally (warnings, upcoming games, ban status, money drops).
+  - Acts as a central point for systems to communicate about game events and state changes. For instance, the Orchestrator may notify the game manager when a player exits a game.
+  - Provides high-level insights on the bot's activity.
+  - Can be persistent, but not at the db level. It only monitors the current state of every game being played.
+  - Logs these insights for monitoring purposes or specialized logging.
+  """
+  def __init__(self):
+    # dict of games, key = chat_id, value = Game object
+    self.games: Dict[str, Game] = {}
```

Comment:
```suggestion
    self.games: dict[str, Game] = {}
```
since python3.10, you can use `dict` as type